client/src/components/getContestComponents/Tabs.js

```diff
@@ -73,16 +77,31 @@ export default function SimpleTabs({ data }) {
         <TabPanel value={value} index={0}>
           {data.submissions ? (
             <GridList cellHeight={160} className={classes.gridList} cols={3}>
-              {data.submissions &&
-                data.submissions.map((pic, i) => (
-                  <GridListTile
-                    key={i}
-                    cols={1}
-                    className={classes.gridListTile}
-                  >
-                    <img src={pic} alt={pic} />
-                  </GridListTile>
-                ))}
+              {data.user && data.user._id === user._id
```

Comment:
we might want to move this logic into the backend, but we can do that after presentation

Reply:
Yes I should. That was some messy ternary. Thanks.